ddtrace/tracer/stats.go

```diff
@@ -164,9 +170,28 @@ func (c *concentrator) Stop() {
 	}
 	close(c.stop)
 	c.wg.Wait()
+empty:
+	for {
+		select {
+		case s := <-c.In:
+			c.statsd().Incr("datadog.tracer.stats.spans_in", nil, 1)
+			c.add(s)
+		default:
+			break empty
+		}
+	}
+	sp := c.flush(time.Now(), withCurrentBucket)
+	c.send(&sp)
```

Comment:
Since these are always called together, do you think it makes sense to absorb the code from `send` into `flush`?

Reply:
I'm fine with merging them into a single function 👍